docs/aws.rst

```diff
@@ -20,8 +20,56 @@ S3
 Overview
 ========
 
-The S3 class heavily reliant on the ``boto3`` python package. It includes a suite of common methods that are commonly
-used with S3.
+S3 is Amazon Web Service's object storage service that allows users to store and access data objects. The Parson's class is a high level wrapper of the AWS SDK `boto3 <https://boto3.amazonaws.com/v1/documentation/api/latest/index.html>`_. It allows users to upload and download files from S3 as well as manipulate buckets.
+
+.. note::
+  Authentication
+    Access to S3 is controlled through AWS Identity and Access Management (IAM) users. Users can be granted granular access to AWS resources, including S3. IAM users are provisioned keys, which are required to access the S3 class. 
```

Comment:
Is there a useful link we can place here, to help users access the AWS IAM interface more quickly?

Reply:
Good suggestion!